### PR TITLE
DOC-3080 TAS connector

### PR DESCRIPTION
**Pull request by roshnisarangadharan**
# Harness Developer Pull Request
Thanks for helping us make the Developer Hub better. The PR will be looked at
by the maintainers. 

## What Type of PR is This?

- [ ] Issue
- [ ] Feature
- [ ] Maintenance/Chore

If tied to an Issue, list the Issue(s) here:

* *Issue(s)*

## House Keeping
Some items to keep track of. Screen shots of changes are optional but would help the maintainers review quicker. 

- [ ] Tested Locally
- [ ] *Optional* Screen Shoot.

### CODE REVIEW

**Review by deepakbolangady — CHANGES_REQUESTED** [review]
The preview environment did not open for me, so that part of my review is pending. I'll take a look again after you've incorporated the comments. Please reach out to me if you have any questions about my comments. Thank you.

File: docs/platform/7_Connectors/Cloud-providers/add-tas-connector.md [hunk 1]
```diff
@@ -0,0 +1,188 @@
...
+    # verify plugins
+    cf plugins
+   ```
+  
+```mdx-code-block
+</TabItem>    
+</Tabs>
+```
+   
+4. Apply the profile to the delegate profile and check the logs.
```
[the remark this answers is not in the document]
Comment (roshnisarangadharan) [reply docs/platform/7_Connectors/Cloud-providers/add-tas-connector.md:138]: we are applying the profile with cf cli commands to the delegate profile

File: docs/platform/7_Connectors/Cloud-providers/add-tas-connector.md [hunk 2]
```diff
@@ -0,0 +1,188 @@
...
+Use this [delegate installation wizard video](https://www.youtube.com/watch?v=yLMCxs3onH8) to guide you through the process.
+
+</details>
+
+```mdx-code-block
+import DelegateInstall from '/tutorials/platform/install-delegate.md';
+```
+
+<details>
+<summary>Install a delegate by using the terminal</summary>
```
Comment (roshnisarangadharan) [inline docs/platform/7_Connectors/Cloud-providers/add-tas-connector.md:32]:
```suggestion
<summary>Use the terminal</summary>
```

File: docs/platform/7_Connectors/Cloud-providers/add-tas-connector.md [hunk 3]
```diff
@@ -0,0 +1,188 @@
...
+This topic describes how to set up the Harness Delegate in your TAS environment and add the cloud provider used to connect to your Tanzu cloud for deployment.
+
+## Install the Harness Delegate
+
+Harness Delegate is a service you run in your local network or VPC to connect your artifacts, TAS infrastructure, and any other providers with the Harness Manager.
+
+Expand the following section to learn how to install the Harness Delegate.
+
+<details>
+<summary>Use the delegate installation wizard</summary>
```
[the remark this answers is not in the document]
Comment (roshnisarangadharan) [reply docs/platform/7_Connectors/Cloud-providers/add-tas-connector.md:16]: I think its is understood if we say use the wizard. Also, I have mentioned, expand the sections to learn how to install delegates.

**Pull request merged**